rpc/lib/server/handlers.go

```diff
@@ -538,7 +538,7 @@ func (wsc *wsConnection) writeRoutine() {
 				return
 			}
 		case msg := <-wsc.writeChan:
-			jsonBytes, err := json.Marshal(msg)
+			jsonBytes, err := json.MarshalIndent(msg, "", "  ")
```

Comment:
> Can't do this with websockets.

Why not? You dont need to see the pretty print until it hits a terminal or a webpage. Install a json pretty printer plugin. I do and its way better than just using MarshalIndent (colors and close/open functionality and so on)